roles/jboss_eap/tasks/main.yml

```diff
@@ -9,3 +9,12 @@
       raw: id -u jboss
       register: jboss_eap_id_jboss
       ignore_errors: yes
+      when: '"jboss.eap.jboss-user" in facts_to_collect'
+    - name: check for common install directories
+      raw: test -d "{{ item }}"
+      register: jboss_eap_common_directories
+      ignore_errors: yes
+      with_items:
+        - /var/log/jboss-as
+        - /usr/log/jboss-as
```

Comment:
Can you provide a link to the follow up investigation issue to obtain a more complete list of common install directories?

Reply:
https://github.com/quipucords/rho/issues/349